Providers/nxOMSAutomationWorker/automationworker/3.x/worker/job.py

```diff
@@ -185,6 +188,45 @@ def unload_job(self):
                                     "Unknown",  # TODO(dalbe): fix runbook source
                                     self.runbook.definition_kind_str, self.runbook_data.name, self.job_data.run_on)
         tracer.log_sandbox_job_unloaded(self.job_id)
+    
+    def __process_job_parameters(self):
```

Comment:
Consider using TypeVar to bound on only the supported types in parameters.
https://docs.python.org/3/library/typing.html#typing.TypeVar